lighthouse-core/test/audits/metrics/cumulative-long-queuing-delay-test.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright 2016 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../../../audits/metrics/cumulative-long-queuing-delay.js');
+const assert = require('assert');
+const options = Audit.defaultOptions;
+
+const pwaTrace = require('../../fixtures/traces/progressive-app-m60.json');
+
+function generateArtifactsWithTrace(trace) {
+  return {
+    traces: {[Audit.DEFAULT_PASS]: trace},
+    devtoolsLogs: {[Audit.DEFAULT_PASS]: []},
+  };
+}
+/* eslint-env jest */
+
+describe('Performance: cumulative-long-queuing-delay audit', () => {
+  it('evaluates cumulative long queuing delay metric properly', () => {
+    const artifacts = generateArtifactsWithTrace(pwaTrace);
+    const settings = {throttlingMethod: 'provided'};
+    const context = {options, settings, computedCache: new Map()};
+
+    return Audit.audit(artifacts, context).then(output => {
```

Comment:
nit: `async/await` and `expect` for new tests

though @brendankenny do you disagree with using `expect` for all new tests?

Reply:
> though @brendankenny do you disagree with using expect for all new tests?

no, it sounds good to me. I've (mostly) been matching the `assert` or `expect` used in existing test files just because I personally don't like when they're mixed, but totally 👍👍 for `expect` in new test files